docs/configuration.md

```diff
@@ -0,0 +1,65 @@
+# Configuration
+
+Mutant can be configured with a config file that can be named one of the following: `.mutant.yml`, `config/mutant.yml`, or `mutant.yml`
+
+The following options can be configured through the config file:
+
+#### `includes`
+
+The includes key takes a list of strings to add to ruby's `$LOAD_PATH`. This is similar to ruby's `-I` flag.
+
+```yml
+---
+includes:
+  - lib
+```
+
+Additional includes can be added by providing the `-I` or `--include` flag to the CLI.
+
+#### `requires`
+
+The requires key takes a list of ruby files to be required. This is how mutant loads your application code.
+
+```yml
+---
+requires:
+  - my_app
+```
+
+Additional requires can be added by providing the `-r` or `--require` flag to the CLI.
+
+#### `integration`
+
+Specifies which mutant integration to use. If your tests are writen in [RSpec](https://rspec.info/), this should be set to `rspec`. If your tests are written in [minitest](https://github.com/seattlerb/minitest), this should be set to `minitest`.
+
+```yml
+---
+integration: rspec
+```
+
+The integration can be overridden by providing the `--use` flag to the CLI.
+
+#### `fail_fast`
+
+When `fail_fast` is enabled, mutant will stop as soon as it encounters an alive mutation. This can be helpful for incremental workflows where you want to see the output of a failed mutation immediately. Valid values are `true` and `false`.
+
+```yml
+---
+fail_fast: true
+```
+
+`fail_fast` cannot currently be overridden by the CLI.
```

Comment:
mhh, this is not my intention, likely a bug I've to fix.

Reply:
it can, but it can only be turned from `false` in the config file (which is semantically redundant, as its the default anyway) to `true` in the CLI.

Reply:
> it can, but it can only be turned from false in the config file (which is semantically redundant, as its the default anyway) to true in the CLI.

Is this how you intend it to work? Or should there be a `--fail-fast false` or `--no-fail-fast` support? I realized this when I was showing @gorgonshead how it worked. You're right that it can go from `false` -> `true` though.

Reply:
@dgollahon I think that we'd want to support `--no-fail-fast` to flip it back. For the moment this PR should state: 

> `fail_fast: true` in the config file cannot be undone in the CLI